devbin/hail-build-llvm

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+cd $LLVM_DIRECTORY
+git checkout llvmorg-15.0.3
+rm -rf $LLVM_BUILD_DIRECTORY
+mkdir -p $LLVM_BUILD_DIRECTORY
+cd $LLVM_BUILD_DIRECTORY
+cmake ../llvm -G Ninja \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_C_COMPILER=clang \
+  -DCMAKE_CXX_COMPILER=clang++ \
+  -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
+  # Uncomment to build examples
+  # -DLLVM_BUILD_EXAMPLES=ON \
+  -DLLVM_ENABLE_ASSERTIONS=ON \
+  -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;mlir" \
+  # Uncomment for a faster build on Linux
+  # -DLLVM_ENABLE_LLD=ON \
```

Comment:
Does this work? The comments break the line continuations. I think this will fail as written. I'd probably do this like:


```
# Uncomment to build examples
# CMAKE_EXTRA_ARGS="${CMAKE_EXTRA_ARGS} -DLLVM_BUILD_EXAMPLES=ON"
# Uncomment for a faster build on Linux
# CMAKE_EXTRA_ARGS="${CMAKE_EXTRA_ARGS} -DLLVM_ENABLE_LLD=ON"
make ../llvm -G Ninja \
   ... \
    ${CMAKE_EXTRA_ARGS}
```

Reply:
Good catch, fixed!